### PR TITLE
Feature/capability

### PR DESCRIPTION
**Pull request by gordonkelly26**
User Stories 10 and 11, simply pulling capability data from the database, their lead info and their job families. Simply displays the data in two tables.

Many thanks,
_Gordon and Dilly_

### CODE REVIEW

File: kjourney/src/app/data.service.ts [hunk 1]
```diff
@@ -12,4 +14,12 @@ export class DataService {
   public getRoles(): Promise<Role[]> {
     return this.http.get<Role[]>('api/roles').toPromise()
   }
+
+  public getCapabilities(): Promise<Capability[]> {
```
Comment (micalexis) [inline kjourney/src/app/data.service.ts]: the use of `Promises` provides no additional benefit and additional computation converting an observable to a promise 

can you revert to using Observables and subscribe to the observables in the components which inject the DataService

File: kjourney/src/app/main/main.component.ts [hunk 2]
```diff
@@ -11,13 +13,17 @@ import { Role } from '../../models/role'
...
+  public capabilities: Capability[]
+  public jobFamilies: JobFamily[]
 
   constructor(data: DataService) {
     this.data = data
   }
 
   async ngOnInit() {
     this.roles = await this.data.getRoles()
+    this.capabilities = await this.data.getCapabilities()
```
Comment (micalexis) [inline kjourney/src/app/main/main.component.ts]: change this to subscribe to the observables from the DataService instead

**Review by micalexis — CHANGES_REQUESTED** [review]
address the comments and squash your commits

File: kjourney/src/app/main/main.component.ts (hunk 2 above)
Comment (Conormc98) [reply kjourney/src/app/main/main.component.ts]: I second this

Comment (Conormc98) [inline kjourney/src/app/main/main.component.ts]: Squash please

**Comment by gordonkelly26** [comment]
Promises changed to observables and commits squashed

**Review by Conormc98 — APPROVED** [review]

**Review by micalexis — APPROVED** [review]
👍

**Review by micalexis — APPROVED** [review]
👍

**Comment by ChrisThompsonK** [comment]
Hi @micalexis @Conormc98 @e-mccoy, this feature is now ready for merging following rebase and squash.

**Review by micalexis — APPROVED** [review]

**Review by ChrisThompsonK — APPROVED** [review]

**Pull request merged**